clintfidel/WEconnect
# #157213223- Users View and Edit their Profile

#### What does this PR do?
This PR enables user view and edit profile

#### Description of Task to be completed?
This PR ensures registererd users are able to view their profile and also edit it
#### How should this be manually tested?
- visit application url [here](weconnectapp.herokuapp.com)
- Navigate to the signup page and create an account or Login if you already have an account
- You would redirected to a page showing a list of your business
- click on the navigation link that shows profile, a dropdown would appear click on my profile, then you should be redirected to the page that shows your profile

- on that page an edit button would be seen, click on the button 
- A modal would pop-up for update, update any field and save
- Your profile page would show the newly updated profile 

#### What are the relevant pivotal tracker stories?
[157213223](https://www.pivotaltracker.com/story/show/157213223)

#### Screen Shot


#### Any background context you want to add?
N/A
